apisix/cli/ops.lua

```diff
@@ -792,10 +803,8 @@ local function start(env, ...)
     end
 
     -- check running
-    local pid_path = env.apisix_home .. "/logs/nginx.pid"
-    local pid = util.read_file(pid_path)
-    pid = tonumber(pid)
-    if pid then
+    local running, pid = check_running(env)
```

Comment:
I think you don't need to `check_running` two times

Reply:
i don't know what you mean check_running 2 times,  this is check running and then do something

Reply:
you check_running in `wait_stop_finish` and `start`, we can simplify this,  just call `check_running` in `start`, and wait in `check_running`

Reply:
but here are different logic, `start` check_running if get  true , will return error and do  not wait .

Reply:
t/cli/test_cmd.sh
```
# check running when run repeatedly
out=$(make run; make run || true)
if ! echo "$out" | grep "the old APISIX is still running"; then
    echo "failed: should find APISIX running"
    exit 1
fi


```

Reply:
we can wait in the `check_running`, return true if the old process exits successfully, return false if reach the deadline

Reply:
ok , i get you point , i fixed  it .